.editorconfig

```diff
@@ -1,7 +1,9 @@
 root = true
 
-[*.{json,js,ts,md,yml}]
+[*]
 insert_final_newline = true
+end_of_line = lf
 
-[.gitignore]
-insert_final_newline = true
+[*.{json,js,ts,md,yml}]
```

Comment:
for json we supposed to use 2: https://github.com/theia-ide/theia/blob/7c678cc13c0c465d0e9551cfde6cd48e29dc79a6/.vscode/settings.json#L44-L46

Reply:
yml also should be 2